bin/ham-bash-lib.sh

```diff
@@ -393,6 +393,72 @@ toolset_import_list() {
     done
 }
 
+# This will always override the MSVC_IDE_DIR.
+# NOTE: This is quite slow and should be used only once during setup.
+toolset_nt_find_ide_dir() {
```

Comment:
This is looking only for MSVC, so probably just `toolset_msvc_find_ide_dir()` makes more sense here.